xml/System.Net.Http/HttpClient.xml

```diff
@@ -85,6 +58,13 @@ public class GoodController : ApiController
     End Sub
 End Class
 ```
+Additional options can be configured by passing in a <xref:System.Net.Http.HttpClientHandler> (or <xref:System.Net.Http.SocketsHttpHandler> in .NET Core 2.1 or later) as part of the constructor. The connection properties on the handler cannot be changed once a request has been submitted, so one reason to create a new HttpClient instance would be if the connection properties need to be changed.
```

Comment:
```suggestion
Additional options can be configured by passing in a "handler", such as <xref:System.Net.Http.HttpClientHandler> (or <xref:System.Net.Http.SocketsHttpHandler> in .NET Core 2.1 or later) as part of the constructor. The properties on the handler cannot be changed once a request has been submitted, so one reason to create a new HttpClient instance would be if the connection properties need to be changed.  If different requests require different settings, this may also lead to an application having multiple <xref:System.Net.Http.HttpClient> instances, where each instance is configured appropriately, and then requests are issued on the relevant client.
```